cost/azure/savings_plan/utilization/azure_savings_plan_utilization.pt

```diff
@@ -0,0 +1,501 @@
+name "Azure Savings Plan Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when Azure Savings Plan Utilizations are identified. NOTE: These Utilizations are generated by Azure. See the [README](https://github.com/flexera-public/policy_templates/tree/master/cost/azure/savings_plan/utilization) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "medium"
+category "Cost"
+default_frequency "monthly"
+info(
+  version: "0.2.0",
+  provider: "Azure",
+  service: "Billing Benefits",
```

Comment:
Should be "Compute", similar to other policies involving SPs.

(I know it's a bit confusing but in the info() block, service is more like category or resource type)